packages/web3-eth-accounts/test/fixtures/account.ts

```diff
@@ -24,8 +24,82 @@ export const validPrivateKeytoAccountData: [string, any][] = [
 ];
 
 export const invalidPrivateKeytoAccountData: [any, string][] = [
```

Comment:
Instead of providing just the error string here, you should provide the actual error expected to be thrown ([example](https://github.com/ChainSafe/web3.js/blob/4.x/packages/web3-utils/test/fixtures/validation.ts#L100))

Reply:
this test and is not part of this PR, please add new issue for improvements in existing approved PRs

Reply:
https://github.com/ChainSafe/web3.js/issues/4637